src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/DependenciesNodeCommandGroupHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies
+{
+    /// <summary>
+    /// Command handler that provides special handling for standard commands on dependency nodes.
+    /// </summary>
+    /// <remarks>
+    /// CPS provides a handler for the same commands, however it does not know anything special about dependency nodes.
+    /// </remarks>
+    [ExportCommandGroup(VSConstants.CMDSETID.StandardCommandSet2K_string)]
+    [AppliesTo(ProjectCapabilities.Cps)]
+    [Order(10)]
+    internal sealed class DependenciesNodeCommandGroupHandler : ICommandGroupHandler
+    {
+        private static readonly CommandStatusResult s_suppressedResult = new CommandStatusResult(handled: true, null, CommandStatus.NotSupported | CommandStatus.Invisible);
+
+        public CommandStatusResult GetCommandStatus(IImmutableSet<IProjectTree> nodes, long commandId, bool focused, string commandText, CommandStatus progressiveStatus)
+        {
+            var cmdId = (VSConstants.VSStd2KCmdID)commandId;
+
+            switch (cmdId)
+            {
+                case VSConstants.VSStd2KCmdID.QUICKOBJECTSEARCH: // Open in Object Browser
+                {
+                    if (nodes.Count == 1)
```

Comment:
Why do we only disable command if a single node is selected?

Reply:
This mirrors CPS's implementation which only enables "Open in Object Browser" when you have a single item selected, presumably to avoid the issues around loading multiple items into the Object Browser.

Reply:
Are we sure this isn't ECMD_VIEWREFINOBJECTBROWSER?

Reply:
That's legacy.

Reply:
Ah, weirdly the managed value is named differently to its native equivalent, they are the same value.